pytest/tests/sanity/concurrent_function_calls.py

```diff
@@ -50,14 +53,33 @@ def process():
     for i in range(100):
         key = bytearray(8)
         key[0] = i % 10
-        res = nodes[4].call_function(
```

Comment:
could we change the call_function implementation to accommodate your fix or is this fix specific to this test?

Reply:
fix is specific to this test due to the way requests are parallelized